src/envo/envo-edit.owl

```diff
@@ -27653,15 +27653,19 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02500044> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03000000> (névé)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000000> "Young granular snow which has undergone melting refreezing and compaction processes")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000000> "Snow which has undergone several cycles of melting and refreezing while being compacted.")
```

Comment:
Better! But we need more insight on what the properties of névé actually are. Many forms of ice have gone through melt-refreeze cycles, I assume not all are called névé. Please open an issue on névé so we can track discussions there.